inflection/tools/dictionary-parser/src/main/resources/org/unicode/wikidata/P898.properties

```diff
@@ -0,0 +1,5 @@
+# Copyright 2025 Unicode Incorporated and others. All rights reserved.
+consonant-end=[^ ˈ'ˌ/\\[\\]iyɨʉɯuɪʏʊeøɘɵɤoe̞ø̞əɤ̞o̞ɛœɜɞʌɔæɐaɶäɑɒ][ ˈ'ˌ/\\[\\]]*$
+consonant-start=^[ ˈ'ˌ/\\[\\]]*[^ ˈ'ˌ/\\[\\]iyɨʉɯuɪʏʊeøɘɵɤoe̞ø̞əɤ̞o̞ɛœɜɞʌɔæɐaɶäɑɒ]
+vowel-end=[iyɨʉɯuɪʏʊeøɘɵɤoe̞ø̞əɤ̞o̞ɛœɜɞʌɔæɐaɶäɑɒ][ ˈ'ˌ/\\[\\]]*$
+vowel-start=^[ ˈ'ˌ/\\[\\]]*[iyɨʉɯuɪʏʊeøɘɵɤoe̞ø̞əɤ̞o̞ɛœɜɞʌɔæɐaɶäɑɒ]
```

Comment:
This is how we extract phonetic information for the 4 most common types. There may be other ways to get the phonetic information. It's likely that this information can also be extended to the phonetic information needed for Turkish (front-round, front-unround, back-round, back-unround, hard-consonant, ...), Korean (rieul-end), and probably others. It's not extracted by default. It can be extracted if the `--add-sound` option is used, which is important for English, French, Italian and others.